assets/js/apps/search/components/filters/text-filter/TextFilter.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import Form from 'react-bootstrap/Form';
+import InputGroup from 'react-bootstrap/InputGroup';
+import FormControl from 'react-bootstrap/FormControl';
+import Button from 'react-bootstrap/Button';
+import PropTypes from 'prop-types';
+
+const TextFilter = ({value,options,onValueChange}) => { 
```

Comment:
Codacy has a fix for the issue: [A space is required after ','.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
const TextFilter = ({value, options,onValueChange}) => { 
```